README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# Seth Harrison - Homework 1
+
+![My Picture](https://media.licdn.com/dms/image/C4D03AQH1WJJtAIa8WQ/profile-displayphoto-shrink_200_200/0?e=1575504000&v=beta&t=RRAgSATjoEZQGGeqapRvpcYsCgMOqM4rRboDfN3fXEQ)
+
+## Education
+
+I am a masters student in the *[Committee on International Relations](https://cir.uchicago.edu/)*. I was at the George Washington University for my undergraduate degree, where I studied:
+
+* **International Relations**
+
+* **Geographic Information Systems**
+
+* **Arabic Language**
```

Comment:
It would be great to know more here. A few lines would really help fill out the bio.